src/server/non-land-grants/pigs-might-fly/mappers/state-to-gas-pigs-mapper.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
did the generic gas output not work here?

Reply:
are you referring to state-to-gas-answers-mapper.js or another output ?